app/src/docker/docker-compose.yml

```diff
@@ -141,8 +143,8 @@ services:
       RABBITMQ_PLACEHOLDERS_USERPASSWORD: guest
       REDIS_PLACEHOLDERS_HOST: redis1
     depends_on:
-      - rabbitmq1
-      - redis1
```

Comment:
Can you just add the `redis1` condition back? Other than that, good to go

Reply:
I was having trouble with the syntax there, it did not like mixing the dashes with update direct reference

Reply:
Do  
    redis1:
        condition: service_started